types/events.go

```diff
@@ -164,10 +164,26 @@ func (se StringEvents) String() string {
 	return strings.TrimRight(sb.String(), "\n")
 }
 
+// Flatten returns a flattened version of StringEvents by grouping all attributes
+// per unique event type.
+func (se StringEvents) Flatten() StringEvents {
```

Comment:
Couldn't this be a private method?

Reply:
Sure, but more importantly it'll most likely be useful for any downstream dev wishing to work with stringified events.